.github/workflows/xrt_ci.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
   
 jobs:
   authorize:
-    runs-on: Ubuntu-22.04
+    runs-on: [self-hosted, Ubuntu-22.04]
```

Comment:
@venkatp-xilinx @MattSnow-amd Could we not use Ubuntu-22.04 on the self hosted machines, since this is conflicting with the public Github cloud runnners